css/main.css

```diff
@@ -552,45 +550,40 @@ hr {
   align-items: center;
   padding: 60px 20px;
   margin: 10px;
-
 }
 
 .bg-1 {
-  background:linear-gradient(0deg, rgb(236, 82, 66,0.8) , rgb(236, 82, 66,0.8)), url('../images/conference.jpg');
-  background-size:cover;
-
-
+  background: linear-gradient(0deg, rgb(236, 82, 66, 0.8), rgb(236, 82, 66, 0.8)), url('../images/conference.jpg');
+  background-size: cover;
 }
 
 .bg-2 {
-  background:linear-gradient(0deg, rgb(236, 82, 66,0.8) , rgb(236, 82, 66,0.8)), url('../images/conference2.jpg');
-  background-size:cover;
+  background: linear-gradient(0deg, rgb(236, 82, 66, 0.8), rgb(236, 82, 66, 0.8)), url('../images/conference2.jpg');
+  background-size: cover;
 }
 
-.single-event h2,.single-event p {
+.single-event h2 {
   font-weight: 700;
   color: white;
 }
 
-
 @media (max-width: 768px) {
   .desktop-menu {
     display: none;
   }
 
-
-  .hamburger-menu-icon{
+  .hamburger-menu-icon {
     display: flex;
     position: fixed;
     top: 10%;
     left: 10%;
-    padding-bottom:20%;
-    background:  transparent;
-    color:#272a31
+    padding-bottom: 20%;
+    background: transparent;
+    color: #272a31;
```

Comment:
- [ ] Please know that the hamburger menu icon when scrolling it blocks the text, I think you should have a top bar fixed instead of fixing only the hamburger menu icon, kindly fix that as well. ☺️